app/lib/features/chat_ng/widgets/chat_editor/chat_editor_actions_preview.dart

```diff
@@ -89,22 +88,20 @@ class ChatEditorActionsPreview extends ConsumerWidget {
         GestureDetector(
           key: closePreviewKey,
           onTap: () async {
-            final isEdit = ref.read(chatEditorStateProvider).isEditing;
             final notifier = ref.read(chatEditorStateProvider.notifier);
-
-            notifier.unsetActions();
+            final isEdit = ref.read(chatEditorStateProvider).isEditing;
             if (isEdit) {
               textEditorState.clear();
-            } else {
-              final body = textEditorState.intoMarkdown();
-              final bodyHtml = textEditorState.intoHtml();
-
-              await saveMsgDraft(body, bodyHtml, roomId, ref);
```

Comment:
removing this entirely, because manual save isn't required where transactional changes below are enough to trigger `transactionStream` listener to update draft state.